message_ix_models/model/water/data/water_for_ppl.py

```diff
@@ -181,8 +180,7 @@ def apply_act_cap_multiplier(
         df = df.merge(cap_fact_parent, how="left")
         df["value"] *= df["cap_fact"] * 1.2  # flexibility
         df.drop(columns="cap_fact", inplace=True)
-    # remove if there are Nan values, but write a log that inform on the parameter and
-    # the head of the data
+    # remove if there are Nan values, but write a log that inform on the parameter and the head of the data
```

Comment:
I think here and in all comments, it was the policy to keep also commented text within the black limits of 88 characters. Although black might not detect it automatically. At least I was asked to do to in previous PRs :)